x11-misc/slim/slim-1.3.9-r1.ebuild

```diff
@@ -0,0 +1,122 @@
+# Copyright 1999-2023 Gentoo Authors
```

Comment:
I had a 1.3.9 locally and posted it on the bugzilla (#832562) before spotting a minor issue that I felt was worth patching. As that ebuild had been made public and offered for inclusion in the tree, I wanted to avoid confusion.

Reply:
bugzilla attachments are irrelevant for portage ebuild revisions.